src/server/pkg/worker/master.go

```diff
@@ -112,7 +113,7 @@ func (a *APIServer) jobSpawner(ctx context.Context, logger *taggedLogger) error
 	if err != nil {
 		return err
 	}
-	pool, err := pool.NewPool(a.kubeClient, a.namespace, ppsserver.PipelineRcName(a.pipelineInfo.Pipeline.Name, a.pipelineInfo.Version), numWorkers, client.PachDialOptions()...)
+	pool, err := pool.NewPool(a.kubeClient, a.namespace, ppsserver.PipelineRcName(a.pipelineInfo.Pipeline.Name, a.pipelineInfo.Version), numWorkers*5, client.PachDialOptions()...)
```

Comment:
Actually we can do something even better which is make this number go away all together.